src/test/java/by/exadel/internship/formTest/FormTest.java

```diff
@@ -46,18 +46,15 @@ private MockMultipartFile formData() {
     public void givenFormWithFile_checkTestData()
             throws Exception {
 
-        MockMultipartFile file = new MockMultipartFile("file", "text.txt",
-                MediaType.MULTIPART_FORM_DATA_VALUE, "some file".getBytes());
-
```

Comment:
Why have u deleted it, i think its better to comment it for the future

Reply:
I got wrong point. I've  restored and commented this